src/Normalizer/ContentEntityNormalizer.php

```diff
@@ -308,14 +300,16 @@ public function denormalize($data, $class, $format = NULL, array $context = [])
       }
     }
 
-    foreach ($site_languages as $site_language) {
-      $langcode = $site_language->getId();
-      if ($entity->language()->getId() != $langcode && isset($translations[$langcode])) {
-        $entity->addTranslation($langcode, $translations[$langcode]);
+    if ($entity instanceof ContentEntityBase) {
```

Comment:
Won't the normalizer only get content entity classes? or is one of the sub classes using this as a base where this is not desired? If we need it, we should check for the appropriate interface instead maybe? Does it need `ContentEntityInterface` or just `TranslatableInterface` or something etc...